crates/kobold/js/util.js

```diff
@@ -22,22 +17,16 @@ export function __kobold_fragment_unmount(f)
 {
 	let [b, e] = fragmentDecorators.get(f);
 	while (b.nextSibling !== e) f.appendChild(b.nextSibling);
-	f.appendChild(e);
-	f.insertBefore(b, f.firstChild);
+	f.appendChild(e).insertBefore(b, f.firstChild);
```

Comment:
`appendChild` returns the child appended, not the parent, this is now equivalent to:

```js
f.appendChild(e);
e.insertBefore(b, f.firstChild);
```
Which will also break